src/components/NavBar/navBar-style.css

```diff
@@ -0,0 +1,8 @@
+.active {
```

Comment:
Is `.active` being used or is that built into the NavLink component?

Reply:
.active belong to component NavLink. We used exact: bool.